src/convertToObject.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString
```

Comment:
The variable `array` is correctly initialized by splitting the `sourceString`. Ensure that `sourceString` is a valid string before calling `convertToObject` to avoid runtime errors.